cibyl/sources/jenkins.py

```diff
@@ -167,7 +171,8 @@ def __init__(self, url: str, username: str = None, token: str = None,
 
     @safe_request
     def send_request(self, query, timeout=None, item="",
-                     api_entrypoint="/api/json", raw_response=False):
+                     api_entrypoint="/api/json", raw_response=False,
+                     url_to_use=None):
```

Comment:
How about calling it simply "url"? I understand that generate_query_url uses it, but that's a different namespace

Reply:
Makes sense, I changed it